### PR TITLE
content: publish practical zephyr device tree

### PR DESCRIPTION
**Pull request by bahildebrand**

### CODE REVIEW

**Comment by bahildebrand** [comment]
@lmapii I made some grammar edits. Let me know what you think

File: _posts/2024-01-31-practical_zephyr_dt.md [hunk 1]
```diff
@@ -9,36 +9,36 @@ In the [previous article]({% post_url 2024-01-24-practical_zephyr_kconfig %}), w
 
 <!-- excerpt start -->
 
-In this third article of the "Practical Zephyr" series we'll see how we configure and use _hardware_. For this, Zephyr borrows another tool from the Linux kernel: [**Devicetree**](https://docs.zephyrproject.org/latest/build/dts/index.html).
+In this third article of the "Practical Zephyr" series, we'll see how we configure and use _hardware_. For this, Zephyr borrows another tool from the Linux kernel: [**Devicetree**](https://docs.zephyrproject.org/latest/build/dts/index.html).
 
-In contrast to _Kconfig_, the *Devicetree* syntax and its use are a bit more intricate. Therefore, we'll cover *Devicetree* in two articles: In this article, we'll see what a Devicetree is and how we can write our own Devicetree source files. In the next article, we'll look at so-called Devicetree _bindings_, which add semantics to our Devicetree. Be prepared for a fair bit of theory, but as usual, we'll use an example project to follow along.
```
Comment (lmapii) [inline _posts/2024-01-31-practical_zephyr_dt.md:14]: For my own information (no pun intended): Does it really sound weird in English to write "write our _own_ source files"? This also popped up in my Grammarly pass, but I really wanted to say that we create our "own" files that are not part of Zephyr, etc.

File: _posts/2024-01-31-practical_zephyr_dt.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ In case you're looking for a more detailed description of the _DTS (Devicetree S
 
 ## Devicetree and Zephyr
 
-To get started, we create a new freestanding application with the files listed below. In case you're not familiar with the required files, the installation, or the build process, have a look at the previous articles or the official documentation. As mentioned in the [pre-requisites](#prerequisites), you should be familiar with creating, building, and running a Zephr application.
+We create a new freestanding application with the files listed below to get started. If you're not familiar with the required files, the installation, or the build process, have a look at the previous articles or the official documentation. As mentioned in the [prerequisites](# prerequisites), you should be familiar with creating, building, and running a Zephr application.
```
Comment (lmapii) [inline _posts/2024-01-31-practical_zephyr_dt.md:80]:
```suggestion
We create a new freestanding application with the files listed below to get started. If you're not familiar with the required files, the installation, or the build process, have a look at the previous articles or the official documentation. As mentioned in the [prerequisites](#prerequisites), you should be familiar with creating, building, and running a Zephr application.
```

**Comment by lmapii** [comment]
Thanks @bahildebrand! Two minor comments and one fix for a broken link, rest looks good to me.

File: _posts/2024-01-31-practical_zephyr_dt.md [hunk 3]
```diff
@@ -60,11 +60,11 @@ The following is a snippet of [Nordic's](https://www.nordicsemi.com/) _Devicetre
...
 
-Instead of going into great detail about the _DTS (Devicetree Source) format_, we'll start with a simple project, build it, and dive straight into the input and output files used or generated by the build process. Based on those files, one by one, we'll try and figure out how this whole thing works. Finally, we'll also create our own nodes, but we won't fully specify the file format.
+Instead of going into great detail about the _DTS (Devicetree Source) format_, we'll start with a simple project, build it, and dive straight into the input and output files used or generated by the build process. Based on those files, one by one, we'll try and figure out how this whole thing works. Finally, we'll create our own nodes, but we will need to fully specify the file format.
```
[the remark this answers is not in the document]
Comment (bahildebrand) [reply _posts/2024-01-31-practical_zephyr_dt.md:67]: Yeah, that wording is a bit strange now that I look at it again. Will revert.

File: _posts/2024-01-31-practical_zephyr_dt.md (hunk 1 above)
Comment (bahildebrand) [reply _posts/2024-01-31-practical_zephyr_dt.md:14]: In most cases it is seen as redundant, but I agree in this case it adds enough flavor to warrant staying. Will re-add.

**Review by gminn — APPROVED** [review]
🚀

**Comment by lmapii** [comment]
@gminn @bahildebrand something's off, the article exists twice on the blog

**Comment by bahildebrand** [comment]
@lmapii on it now, I used `cp` instead of `mv` when changing the date :facepalm:

**Pull request merged**